app/uk/gov/nationalarchives/omega/editorial/connectors/ApiConnector.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2022 The National Archives
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy of
+ * this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to
+ * use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+ * the Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+ * FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+ * COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+ * IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+ * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+package uk.gov.nationalarchives.omega.editorial.connectors
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import javax.inject.{ Inject, Singleton }
+import org.typelevel.log4cats.SelfAwareStructuredLogger
+import org.typelevel.log4cats.slf4j.Slf4jLogger
+import play.api.inject.ApplicationLifecycle
+import play.api.libs.json.{ Json, Reads }
+
+import uk.gov.nationalarchives.omega.editorial.models.{ EditSet, GetEditSet }
+import uk.gov.nationalarchives.omega.editorial.support.TimeProvider
+import uk.gov.nationalarchives.omega.editorial.config.Config
+
+@Singleton
+class ApiConnector @Inject() (
+  config: Config,
+  timeProvider: TimeProvider,
+  lifecycle: ApplicationLifecycle
+) {
+  import ApiConnector._
+
+  private implicit val logger: SelfAwareStructuredLogger[IO] = Slf4jLogger.getLogger[IO]
+
+  private val getEditSetSID = "OSGEES001"
```

Comment:
I was going to say that this was duplicated from `ResponseBuilder` - but that one belongs to the `StubServer`, which is only used for testing. 

Given this, I wonder if we should move the latter two into the test source tree?

Reply:
Out of curiosity, how would you use the stub server outside of automated tests?

Reply:
The stub server is enabled here, in the default configuration: https://github.com/nationalarchives/ctd-omega-editorial-frontend/blob/main/conf/application.conf#L5

Reply:
Eventually, I see this running against the real backend (aka `ctd-omega-services`).

So I see the stub service as a permanent addition for automated testing, but only a stop gap for manual running.

As such: yes, it should stay where it is.